packages/frontend/core/src/pages/workspace/detail-page/detail-page-header.tsx

```diff
@@ -115,6 +125,16 @@ export function NormalPageHeader({ page, workspace }: PageHeaderProps) {
   const onRename = useCallback(() => {
     setTimeout(() => titleInputHandleRef.current?.triggerEdit());
   }, []);
+
+  const onTriggerEdit = useCallback(() => {
+    mixpanel.track('header option click', {
```

Comment:
the tracked event name should be camel-case, like  `HeaderOptionClick`

Reply:
I also changed the event name in the relevant PR.